src/pages/Home.jsx

```diff
@@ -1,3 +1,48 @@
-const Home = () => <h1>Welcome to Voma</h1>;
+import styled from 'styled-components';
 
-export default Home;
+const StyledSection = styled.section`
+  width: 50%;
+  margin: 0 auto;
+  padding-top: 100px;
+`;
+
+const StyledH1 = styled.h1`
+  font-size: 3rem;
+  font-weight: 600;
+`;
+
+const StyledParagraph = styled.p`
+  font-size: 1rem;
+`;
+
+const SlackButton = styled.button`
+  width: 30%;
```

Comment:
Can you please remove this width? It's making the button all blurry. It's fine if it doesn't match the design completely.

![Screen Shot 2021-07-27 at 10 12 43 AM](https://user-images.githubusercontent.com/9326234/127180967-e0b0e7ab-f30c-43ef-b113-f2cd436d58eb.png)